addons/api/addon/handlers/indexed-db-handler.js

```diff
@@ -64,6 +64,11 @@ export default class IndexedDbHandler {
           if (storeToken) {
             const tokenObj = await indexedDb.token.get(tokenKey);
             listToken = tokenObj?.token;
+          } else {
```

Comment:
This was a bug that got missed when I added batching to requests. It was mistakenly deleting on every batch insert instead of just once the first time so you only ever got the last page of results.